### PR TITLE
[BREAK] Rename QosException subclasses

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

**Comment by uschi2000** [comment]
This is an API break, but I'm willing to take the risk under the assumption that nobody has used this classes yet.

File: errors/src/main/java/com/palantir/remoting/api/errors/QosException.java [hunk 1]
```diff
@@ -51,17 +51,16 @@ public static RetryOther retryOther(URL redirectTo) {
     }
 
     /**
-     * An exception indicating that all nodes of this service are currently unavailable and the client shall not attempt
-     * to wait for it to become available again. Typically, human intervention may be required to bring this service
-     * back up.
+     * An exception indicating that (this node of) this service are currently unavailable and the client may try again
```
Comment (chrisalice) [inline errors/src/main/java/com/palantir/remoting/api/errors/QosException.java]: "is currently"

Comment (uschi2000) [reply errors/src/main/java/com/palantir/remoting/api/errors/QosException.java]: Done here: https://github.com/palantir/http-remoting-api/pull/40

Comment (uschi2000) [reply errors/src/main/java/com/palantir/remoting/api/errors/QosException.java]: Thanks

**Pull request merged**